packages/server/src/oauth/utils.ts

```diff
@@ -777,18 +785,18 @@ function includeRefreshToken(request: LoginRequest): boolean {
 /**
  * Returns the external identity provider user info for an access token.
  * This can be used to verify the access token and get the user's email address.
- * @param idp - The identity provider configuration.
+ * @param userInfoUrl - The user info URL from the identity provider configuration.
  * @param externalAccessToken - The external identity provider access token.
  * @returns The user info claims.
  */
 export async function getExternalUserInfo(
```

Comment:
This came together nicely.  We're using the same external userinfo flow for both RFC 7662 (this PR) and RFC 8693 (OAuth 2.0 Token Exchange).

In the future, this should probably be extended to support both `/introspect` and `/userinfo`.  Token introspection is the better option, but it is not as widely supported as user info.